ddtrace/internal/runtime/runtime_metrics.py

```diff
@@ -86,3 +90,9 @@ def __repr__(self):
             self.__class__.__name__,
             self._runtime_metrics,
         )
+
+    def recreate(self):
```

Comment:
```suggestion
    def copy(self):
```
is more Pythonic.

Reply:
👍🏽

Reply:
I see you changed back, is that a mistake?

Reply:
I thought it best this PR maintain the current writer API, which we are rethinking of course but given #2098 I thought it best not to change this here.

Reply:
Though this is not the writer API, this is the `RuntimeWorker` API and you're adding something. I think it's fair to add the _right_ thing now so we don't have to come back to it later. :)

Reply:
Done!